src/SiteConfig/ConfigBuilder.php

```diff
@@ -412,6 +407,14 @@ public function parseLines(array $lines)
             }
         }
 
+        // in case of bad configuration
+        if (\count($config->find_string) !== \count($config->replace_string)) {
+            $this->logger->warning('find_string & replace_string size mismatch, check the site config to fix it', ['find_string' => $config->find_string, 'replace_string' => $config->replace_string]);
+
+            $config->find_string = [];
+            $config->replace_string = [];
```

Comment:
Maybe we could zip them together (i.e. trim them to the length of the shorter)?

Reply:
Won't we have undesired side effects?

In that case, it'll work
```
find_string: badstuff
replace_string: goodone
replace_string: src="
```

In that case, it'll not work because you'll replace `hellow` by `goodone` if you _zip_ them.
```
find_string: hellow
find_string: badstuff
replace_string: goodone
```

I think it's preferable to empty them and log a warning to the user instead.

Reply:
Either way sounds reasonable to me.